packages/react-components/package.json

```diff
@@ -58,9 +59,10 @@
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "sass": "^1.49.9",
+    "storybook": "^7.0.20",
     "storybook-addon-themes": "^6.1.0",
     "typescript": "^4.5.4",
-    "vite": "^2.8.0",
+    "vite": "^3.0.0",
```

Comment:
also we can check if it's possible to upgrade to a newer version of `vite, cause currently it's 4.3.9